### PR TITLE
fix keepalivemanager

### PR DESCRIPTION
**Pull request by ctwomblyamzn**
*Issue #, if available:*

Internal issue: V410331806

*Description of changes:*

KeepaliveManager was not actually fixed with this commit: https://github.com/amazon-connect/amazon-connect-streams/commit/98e883d65e608d1242b3cb209b88c050c0aabe35. This has been tested and fixes the keepalivemanager termination problem. In doing so, this will ensure that we pop a loginPopup page if the shared worker disconnects for whatever reason (usually that the user is not logged in or their session expired). 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: release/connect-streams.js [hunk 1]
```diff
@@ -25837,7 +25837,7 @@
  
   var CCP_SYN_TIMEOUT = 1000; // 1 sec
   var CCP_ACK_TIMEOUT = 3000; // 3 sec
-  var CCP_LOAD_TIMEOUT = 3000; // 3 sec
+  var CCP_LOAD_TIMEOUT = 10000; // 10 sec
```
Comment (jagadeeshaby) [inline release/connect-streams.js:25840]: not sure why this is showing as 10sec.

**Review by jagadeeshaby — APPROVED** [review]

File: release/connect-streams.js (hunk 1 above)
Comment (ctwomblyamzn) [reply release/connect-streams.js:25840]: Good catch, must not have npm run released.

**Comment by cclifford3** [comment]
Thanks for releasing this! @ctwomblyamzn do you know when it will be published to NPM? 

https://www.npmjs.com/package/amazon-connect-streams

**Comment by ctwomblyamzn** [comment]
The change is now released on npm as well.

**Pull request merged**